### PR TITLE
Startup Script

### PR DESCRIPTION
**Pull request by iammosespaulr**
Added startup Script

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!bin/bash
+cd ..;
+cd mongod;
+gnome-terminal -e "sudo mongod -dbpath .";
```
Comment (malithsen) [inline start.sh:4]: This would only work in environments where gnome is used. A more generalized method is preferred

**Review by malithsen — CHANGES_REQUESTED** [review]

File: start.sh (hunk 1 above)
Comment (charithccmc) [reply start.sh:4]: @iammosespaulr please add changes.

Comment (iammosespaulr) [reply start.sh:4]: on it ... I'll submit a new pr ... and close this in the process

**Comment by ivantha** [comment]
@iammosespaulr Are you still working on this? 🙂

**Comment by iammosespaulr** [comment]
> @iammosespaulr Are you still working on this? 🙂

Well ... Thanks for reminding me @OshanIvantha ( I Forgot ) , I'm gonna write this ...

**Comment by ivantha** [comment]
Duplicate of #244

**Pull request closed without merging**